src/components/ScholarshipForm.jsx

```diff
@@ -78,56 +90,111 @@ function ScholarshipForm({ setSubmissionAlert }) {
-        value={formik.values.deadline}
-        onChange={(date) => formik.setFieldValue('deadline', date, true)}
-      />
+      <Grid item sm={6} xs={12}>
+        <FormikTextField
+          label="Scholarship Name *"
+          id="name"
+          formik={formik}
+          labelStyle={classes.inputLabel}
+        />
+      </Grid>
 
-      <div>
+      <Grid item sm={6} xs={12}>
         <FormikTextField
-          label="Description *"
-          id="description"
+          label="Organization"
+          id="organization"
           formik={formik}
-          rows={8}
+          labelStyle={classes.inputLabel}
         />
-      </div>
+      </Grid>
 
-      <div>
+      <Grid item sm={6} xs={12}>
         <FormikTextField
           label="Scholarship Link *"
           id="website"
           formik={formik}
+          labelStyle={classes.inputLabel}
         />
-      </div>
+      </Grid>
+
+      <Grid item sm={6}>
+        <DatePicker
+          id="deadline"
+          label="Deadline *"
+          labelStyle={classes.inputLabel}
+          error={formik.touched.deadline && Boolean(formik.errors.deadline)}
+          helperText={formik.touched.deadline && formik.errors.deadline}
+          value={formik.values.deadline}
+          onChange={(date) => formik.setFieldValue('deadline', date, true)}
+        />
+      </Grid>
+
+      <Grid item>
+        <ScholarshipAmountField
+          helperText={
+            (getIn(formik.touched, 'amount.type') &&
+              getIn(formik.errors, 'amount.type')) ||
+            getIn(formik.errors, 'amount.min') ||
+            getIn(formik.errors, 'amount.max')
+          }
+          amountType={formik.values.amount.type ?? 'UNKNOWN'}
+          minAmount={formik.values.amount.min}
+          maxAmount={formik.values.amount.max}
+          onTypeChange={(e) =>
+            formik.setFieldValue('amount.type', e.target.value, true)
+          }
+          updateAmount={updateAmount}
+        />
+      </Grid>
 
-      <ScholarshipAmountField
-        className={classes.fieldStyle}
-        helperText={
-          (getIn(formik.touched, 'amount.type') &&
-            getIn(formik.errors, 'amount.type')) ||
-          getIn(formik.errors, 'amount.min') ||
-          getIn(formik.errors, 'amount.max') ||
-          ''
-        }
-        amountType={formik.values.amount.type}
-        minAmount={formik.values.amount.min}
-        maxAmount={formik.values.amount.max}
-        onTypeChange={(e) =>
-          formik.setFieldValue('amount.type', e.target.value, true)
-        }
-        updateAmount={updateAmount}
-      />
+      <Grid item xs={12}>
+        <FormikTextField
+          label="Description *"
+          id="description"
+          labelStyle={classes.inputLabel}
+          formik={formik}
+          rows={8}
+        />
+      </Grid>
+    </Grid>
+  );
+
+  stepperItems.Requirements = 'requirements';
```

Comment:
I'd remove this from the stepper items for now as it's not doing anything. You could add a **Confirmation** step though (maybe in another PR) where you pass the **Scholarship** object to the **ScholarshipDetailCard** component and have the user review/confirm details